src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/DummyNode.cs

```diff
@@ -14,13 +15,18 @@ public void CustomizeView(DummyNode model, Dynamo.Controls.NodeView nodeView)
             if (model.NodeNature == DummyNode.Nature.Unresolved)
                 fileName = "MissingNode.png";
 
+            model.State = ElementState.Warning;
+
             var src = @"/CoreNodeModelsWpf;component/Resources/" + fileName;
 
-            Image dummyNodeImage = new Image()
+            Image dummyNodeImage = new Image
             {
-                Stretch = System.Windows.Media.Stretch.None,
+                Width = 66.0,
+                Height = 66.0,
+                Stretch = Stretch.UniformToFill,
                 Source = new BitmapImage(new Uri(src, UriKind.Relative))
             };
+            RenderOptions.SetBitmapScalingMode(dummyNodeImage, BitmapScalingMode.HighQuality);
```

Comment:
Hmm, odd this is part of a different branch.
This branch is up to date with upstream/master.